pandas/io/excel/_openpyxl.py

```diff
@@ -509,24 +509,14 @@ def get_sheet_by_index(self, index: int):
 
     def _convert_cell(self, cell, convert_float: bool) -> Scalar:
 
-        from openpyxl.cell.cell import TYPE_BOOL, TYPE_ERROR, TYPE_NUMERIC
+        from openpyxl.cell.cell import TYPE_ERROR, TYPE_NUMERIC
 
         if cell.value is None:
             return ""  # compat with xlrd
-        elif cell.is_date:
-            return cell.value
         elif cell.data_type == TYPE_ERROR:
             return np.nan
-        elif cell.data_type == TYPE_BOOL:
-            return bool(cell.value)
```

Comment:
Are there tests for bool/int/etc?

Reply:
Yes there are. If I break the function to return

- all numbers as float, 198 tests fail
- all numbers as int, 211 tests fail
- all bools as int, 20 tests fail
- all bools as str, zero tests fail because TextParser does its own conversions

Thanks for taking them time to look at my first pull request. Let me know if there's anything that needs to be improved.

Reply:
Great - thanks for checking.